config/initializers/rack-attack.rb

```diff
@@ -38,3 +38,23 @@
     [404, {}, ['Not found']]
   end
 end
+
+Rack::Attack.throttled_responder = lambda do |request|
+  match_data = request.env['rack.attack.match_data']
+  now = match_data[:epoch_time]
+  retry_after = match_data[:period] - (now % match_data[:period])
+
+  headers = {
+    'RateLimit-Limit' => match_data[:limit].to_s,
+    'RateLimit-Remaining' => '0',
+    'RateLimit-Reset' => (now + (match_data[:period] - (now % match_data[:period]))).to_s
+  }
+
+  [429, headers, ["Your request has been throttled. Try again in #{retry_after.seconds.from_now}.\n"]]
```

Comment:
`x.seconds.from_now` gives a timestamp. So either we change the message to "try again at" or we just use the seconds